src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -3128,6 +3128,76 @@ def test_aks_nodepool_add_with_gpu_instance_profile(
             checks=[self.is_empty()],
         )
 
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(
+        random_name_length=17, name_prefix="clitest", location="eastus2euap"
```

Comment:
use eastus for the recording tests, we use eastus2euap at last weekend because the RP release 20240331 didn't reach eastus region.

Reply:
updated location to use eastus and did live test, passed